docker/search-api/entrypoint.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -o errexit
 
 # Pass the HOST_UID and HOST_UID from environment variables specified in the child image docker-compose
 HOST_GID=${HOST_GID}
```

Comment:
Could be mistaken, but I think this is a no-op: It's already an envvar, so you're just assigning it to itself.

Reply:
Not what you thought. The HOST_GID and HOST_UID get passed from system env var to docker-compose, then passed to the docker build. And bing then used by the entrypoint script at container startup. It won’t work if not following this workflow.